modules/nf-core/bcftools/mpileup/main.nf

```diff
@@ -55,4 +55,18 @@ process BCFTOOLS_MPILEUP {
         bcftools: \$(bcftools --version 2>&1 | head -n1 | sed 's/^.*bcftools //; s/ .*\$//')
     END_VERSIONS
     """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.bcftools_stats.txt
+    touch ${prefix}.vcf.gz
+    touch ${prefix}.vcf.gz.tbi
+    touch ${prefix}.mpileup.gz
```

Comment:
```suggestion
    echo "" | gzip ${prefix}.vcf.gz
    touch ${prefix}.vcf.gz.tbi
    echo "" | gzip ${prefix}.mpileup.gz
```

This will prevent an error in `nf-test` :)

Reply:
Thanks!